README.md

```diff
@@ -11,6 +11,8 @@ Users are responsible for providing their own VMware software, Packet account, a
 
 The build (with default settings) typically takes 70-75 minutes.
 
+**This repo is by no means meant for production purposes**, a production cluster is possible, but needs some modificatnios. If your desire is to create a productino deployment, please consult with Packet Support via a support ticket.
```

Comment:
Some typos here: `modificatnios` and `productino`

Reply:
@c0dyhi11  can you address the typos and we'll merge?

Reply:
> needs some modifications

it would be more helpful to state the reason why such mods are necessary, and briefly decribe what they might be.

my understand based on discussions with the packet team, is that the reason is that as it stands google-anthos will fail fairly often.  this has been my own experience.

this fact should be disclosed.  better yet, disclose the expected failure rate.  at the moment, i'm seeing maybe 25% failures.  (some attributable to packet provisioning failures, some apparently due to later failures that are not handled by google-anthos.)  what failure rate are others seeing?


i'm not sure what Cody has in mind for the mods, but for my part i intend to implement a retry.

Reply:
I think it would be fair to consider this project to be under active development.  I'm not sure that I would characterize this project as experimental, but I would defer to the experience of folks that have successfully deployed this more than once :-)

Reply:
I corrected the typos and merged this.